gulpfile.js

```diff
@@ -809,6 +810,15 @@ function getRCBranchName() {
   return 'rc_' + yyyy + '_' + mm;
 };
 
+// Helper function: get a minor version number for the day.  Format: yyyymmdd.
+function getNewMinorVersionNumber() {
```

Comment:
This isn't used anywhere, just checking that wasn't accidental.

Reply:
Good point.  Switching to `bump_version` means I stopped using this.